apps/docs/docs/learn/05-theming/02-using-variables.mdx

```diff
@@ -40,6 +40,19 @@ export const colors = stylex.defineVars({
   fontFamily: 'system-ui, sans-serif',
   fontSize: '16px',
 });
+
+const grid = 4;
+const px = (value: string | number) => `${value}px`;
+export const spacing = stylex.defineVars({
```

Comment:
How does adding a second example help explain how `defineVars` works? Please explain the need for this.

Reply:
Adding the `spacing` variable, my intention was to ensure consistency and clarity in our documentation. The code example in _components/MyComponent.js_ references `import {colors, spacing} from '../tokens.styles';` , which implies that both colors and spacing are defined within _tokens.styles.js_ . 

Without the spacing variable defined in _tokens.stylex.js_, this could lead to confusion for users trying to understand the complete implementation of StyleX variables. 

By including the spacing definition, it provides a more comprehensive example and aligns the documentation with the code, enhancing overall clarity and usability.